src/stellar/operations.ts

```diff
@@ -160,8 +160,8 @@ export async function openSellOffer(
       Operation.manageSellOffer({
         selling: newAsset,
         buying: XLM,
-        amount: amount.toString(),
-        price: price.toString(),
+        amount: amount.toFixed(6),
```

Comment:
It's not blocking, but I just wonder what is the motivation behind changing toString() to toFixed(6) 🤔

Reply:
Here's the reason: https://github.com/skyhitz/backend/pull/64
I'm not sure why it's being shown as it was already merged 🤔